ci/jenkins/README.md

```diff
@@ -93,60 +93,62 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY}
 ```
 
-* windows conformance: community tests on Windows cluster with Docker runtime, focusing on "Conformance|sig-windows" and
-  "sig-network", and skipping "LinuxOnly", "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli", "sig-storage",
-  "sig-auth", "sig-api-machinery", "sig-apps", "sig-node", "Privileged", "should be able to change the type from",
-  "[sig-network] Services should be able to create a functioning NodePort service [Conformance]", "Service endpoints
-  latency should not be very high".
+* windows e2e: e2e tests in a Windows setup with Containerd runtime.
 
 ```shell
 #!/bin/bash
 DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
-./ci/jenkins/test.sh --testcase windows-conformance --registry ${DOCKER_REGISTRY}
+./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY}
 ```
 
-* windows network policy: community tests on Windows cluster with Docker runtime, focusing on "Feature:NetworkPolicy".
+* windows conformance: community tests on Windows cluster with Docker runtime, focusing on "Conformance|sig-windows" and
+  "sig-network", and skipping "LinuxOnly", "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli", "sig-storage",
+  "sig-auth", "sig-api-machinery", "sig-apps", "sig-node", "Privileged", "should be able to change the type from",
+  "[sig-network] Services should be able to create a functioning NodePort service [Conformance]", "Service endpoints
+  latency should not be very high".
 
 ```shell
 #!/bin/bash
 DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
-./ci/jenkins/test.sh --testcase windows-networkpolicy --registry ${DOCKER_REGISTRY}
+./ci/jenkins/test.sh --testcase windows-conformance --registry ${DOCKER_REGISTRY}
 ```
 
-* windows e2e with proxyAll enabled: e2e tests in a Windows setup with proxyAll enabled.
+* windows conformance: community tests on Windows cluster with Containerd runtime
+  and OVS running inside container, focusing on "Conformance|sig-windows" and
+  "sig-network", and skipping "LinuxOnly", "Slow", "Serial", "Disruptive", "Flaky", "Feature", "sig-cli", "sig-storage",
+  "sig-auth", "sig-api-machinery", "sig-apps", "sig-node", "Privileged", "should be able to change the type from",
+  "[sig-network] Services should be able to create a functioning NodePort service [Conformance]", "Service endpoints
+  latency should not be very high".
 
 ```shell
 #!/bin/bash
 DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
-./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY} --proxyall
+./ci/jenkins/test.sh --testcase windows-conformance --registry ${DOCKER_REGISTRY}
 ```
 
-* windows containerd e2e: e2e tests in a Windows setup with containerd runtime.
+* windows network policy: community tests on Windows cluster with Docker runtime,focusing on "Feature:NetworkPolicy".
```

Comment:
since ovs processes on windows is running inside containers and docker runtime is no longer supported, there will be only one NetworkPolicy test related to containerd runtime and ovs running in container. 134 and 143 lines are dups.